src/backend/catalog/aclchk.c

```diff
@@ -1581,7 +1581,7 @@ RemoveRoleFromObjectACL(Oid roleid, Oid classid, Oid objid)
 				istmt.objtype = ACL_OBJECT_FDW;
 				break;
 			case ExtprotocolRelationId:
-				istmt.objtype = OBJECT_EXTPROTOCOL;
+				istmt.objtype = ACL_OBJECT_EXTPROTOCOL;
```

Comment:
it is cool, but you should do that during cherry-pick -e, while in one session editor is opened, and you may do changes, git add them and git cherry-pick --continue in another session

Reply:
Sure, amended that commit